src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -152,28 +158,31 @@ public enum Modifier {
              * module</i> to have an implicitly declared dependence on the module
              * named by the {@code Requires}.
              */
-            TRANSITIVE,
+            TRANSITIVE(AccessFlag.TRANSITIVE.mask()),
 
             /**
              * The dependence is mandatory in the static phase, during compilation,
              * but is optional in the dynamic phase, during execution.
              */
-            STATIC,
+            STATIC(AccessFlag.STATIC.mask()),
```

Comment:
This one should have been `AccessFlag.STATIC_PHASE`

Reply:
ok, fix later